yarn.lock

```diff
@@ -9977,16 +9946,11 @@ caniuse-lite@^1.0.0, caniuse-lite@^1.0.30001109:
   resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001335.tgz"
   integrity sha512-ddP1Tgm7z2iIxu6QTtbZUv6HJxSaV/PZeSrWFZtbY4JZ69tOeNhBCl3HyRQgeNZKE5AOn1kpV7fhljigy0Ty3w==
 
-caniuse-lite@^1.0.30001286, caniuse-lite@^1.0.30001335, caniuse-lite@^1.0.30001358, caniuse-lite@^1.0.30001400:
+caniuse-lite@^1.0.30001335, caniuse-lite@^1.0.30001400:
```

Comment:
__question:__ can this also be joined with `caniuse-lite@^1.0.0, caniuse-lite@^1.0.30001109:` above?

Reply:
Great catch! Added here https://github.com/elastic/kibana/pull/161184/commits/4650d2625db6ba9caaedfaa263503de95a7a0aa6